### ISSUE TITLE
Tricaster PRV/PGM Problem

### ISSUE THREAD

**Issue by malot** [josephdadams/TallyArbiter#452]
Hello,

i've set up the TC1 with 4 Inputs and some kind of Wemos D1 (ESP8266) chips and WS2812 LED Stripe as tally lights. For the beginning i work only with program und red light. Until here everthing works right. When i only work with green lights for PRV everything also works fine. 
If i try to use a combination of green light (PRV) and redlight (PGM) most of the things didn't work like expected. 

In the tally arbiter backend is everthing shown correct. But the lights didn't go on/off like expected. 

For Example: 
IN1 -> Prv ->Tally Lights up green                                      1 = GREEN
IN2 -> Pgm -> Tally Lights up red                                       2 = RED

If i put now the **IN1 in PGM** it lights up RED, The Tally from IN2 switches to OFF. Thats fine.
But if i now put IN2 back to PGM. IN2 starts to light up RED.
IN1 only switches to OFF. Normally it should go back green for PRV. But i think the Actions for this Preview isn't fired again after the signal change. 

Is there workaround that we can use PRV and PGM.  If we only use one of these everthing is normal. 

Iv'e use 4 Webhooks für ever Input: 
      PGM: Light RED
NO PGM: Light OFF
      PRV:  Light GREEN
NO PRV: Light OFF

[config.txt](https://github.com/josephdadams/TallyArbiter/files/8806495/config.txt)

Thanks

**Comment by josephdadams** [comment]
Yes this is a known "issue" with using webhooks; it is sending one webhook for the pgm off and another for pvw on, and they arrive out of sequence so you don't always get an accurate state. It's not really fixable when using a webhook to send a command on multiple tally states; you would need to create a separate listener client to mitigate this issue. See #16
